src/pages/documentation/index.md

```diff
@@ -11,9 +11,13 @@ Welcome to the home of Adobe Experience Platform Mobile SDK documentation.
 
 <InlineAlert variant="success" slots="header, text"/>
 
-Good to go for iOS 17
+Good to go for iOS 17 and Android 14
 
-The latest versions of the Adobe Experience Platform SDK for Apple iOS (Swift and Objective-C) are tested for compatibility with the release version of Apple iOS 17. See [Current SDK Versions](./current-sdk-versions.md) to find our latest SDK versions, or [release notes](./release-notes/index.md) for information on changes and updates.
+The latest versions of the Adobe Experience Platform SDK for Android and iOS (Swift and Objective-C) are tested for compatibility with the release versions of Google Android 14 and Apple iOS 17. See [Current SDK Versions](./current-sdk-versions.md) to find our latest SDK versions, or [release notes](./release-notes/index.md) for information on changes and updates.
+
+<InlineAlert variant="info" slots="text"/>
+
+At WWDC23, Apple announced several new privacy features impacting applications and SDKs. The announcement included an upcoming requirement for developers to provide the reason for using certain APIs — including APIs currently used by the AEP Mobile SDK — in the privacy manifest. Apple plans to start enforcing this requirement in the spring of 2024. Adobe is currently working with Apple to better understand the impact of this change. We anticipate that we will be able to provide further information regarding how this change will impact iOS applications that use the AEP Mobile SDK in the coming months.
```

Comment:
Nit that I care about - it shouldn't be AEP Mobile SDK, it should be Adobe Experience Platform Mobile SDK for the first usage, and then Platform Mobile SDK for subsequent usages.